packages/dev/core/src/FlowGraph/flowGraph.ts

```diff
@@ -120,11 +130,11 @@ export class FlowGraph {
         for (const block1 of this._eventBlocks) {
             // If the block is a mesh pick, guarantee that picks of children meshes come before picks of parent meshes
             if (block1.getClassName() === FlowGraphMeshPickEventBlock.ClassName) {
-                const mesh1 = (block1 as FlowGraphMeshPickEventBlock)._getReferencedMesh(context);
+                const mesh1 = (block1 as FlowGraphMeshPickEventBlock)._getReferencedMesh();

```

Comment:
i noticed you have removed the context here. Is it expected to get the object directly from the accessor? Is the accessor part of the context, or are they outside of the graph's context?

Reply:
Yes, the object would be obtained from the accessor. The accessor isn't part of the context, but depending on what accessor I'm using, it could take the context as an argument.